src/main/kotlin/com/ajaxproject/warehouse/entity/MongoWaybill.kt

```diff
@@ -0,0 +1,25 @@
+package com.ajaxproject.warehouse.entity
+
+import org.bson.types.ObjectId
+import org.springframework.data.annotation.Id
+import org.springframework.data.mongodb.core.mapping.Document
+import java.time.LocalDate
+
+@Document(MongoWaybill.COLLECTION_NAME)
+data class MongoWaybill(
+    @Id
+    val id: ObjectId? = null,
+    val customerId: ObjectId,
+    val date: LocalDate,
+    val products: List<MongoWaybillProduct> = listOf()
+) {
+    @Document("products")
+    data class MongoWaybillProduct(
```

Comment:
move it out of the `MongoWaybill` class. It's the separate document. I see no reasons making nested `MongoWaybillProduct`

Reply:
done